stripe/src/test/java/com/stripe/android/paymentsheet/PaymentSheetAddCardFragmentTest.kt

```diff
@@ -22,6 +22,8 @@ import com.stripe.android.paymentsheet.model.FragmentConfigFixtures
 import com.stripe.android.paymentsheet.model.PaymentSelection
 import com.stripe.android.paymentsheet.model.ViewState
 import com.stripe.android.paymentsheet.ui.PaymentSheetFragmentFactory
+import com.stripe.android.utils.TestUtils
+import com.stripe.android.view.Country
```

Comment:
I think one test case could be added for losing focus when no text has been entered yet.  In that case, no error should be displayed.